src/strands/multiagent/base.py

```diff
@@ -137,13 +145,18 @@ def from_dict(cls, data: dict[str, Any]) -> "MultiAgentResult":
         usage = _parse_usage(data.get("accumulated_usage", {}))
         metrics = _parse_metrics(data.get("accumulated_metrics", {}))
 
+        interrupts = []
+        for interrupt_data in data.get("interrupts", []):
```

Comment:
No SessionManager changes needed for this, lovely.

Can you update the PR description indicating that SessionManagement will store/restore interrupts correctly?